streetcrm/admin.py

```diff
@@ -638,15 +655,18 @@ def export_search(request, form, advanced, search_query):
         
         last_header=[]
         header=[]
+        
+        search_title = 'Created: {} - Query: {}'.format(
```

Comment:
Yep, open to changes there, and also want to call out that the only way I'm differentiating tags from other comma separated items is not including a space between them, so that could definitely be improved.